src/testFunctional/resources/migration/V001.1__CREATE_BOOKS_TABLE.sql

```diff
@@ -6,7 +6,7 @@ create table author
 );
 create table books
 (
-    publication_year date,
+    publication_year smallint,
```

Comment:
Once a migration script is deployed it is wrong to modify it, it is also not allowed by flyway and it will break deployment.
Create new script version to make the modification